torch/distributions/utils.py

```diff
@@ -132,15 +132,24 @@ def logits_to_probs(logits, is_binary=False):
     return softmax(logits)
 
 
+def clamp_probs(probs):
+    eps = _finfo(probs).eps
+    return probs.clamp(min=eps, max=1 - eps)
+
+
+def clamp_logits(logits):
+    eps = _finfo(logits).eps
+    return logits.clamp(min=math.log(eps), max=math.log1p(-eps))
```

Comment:
Have you checked that `log`/`log1p` are accurate in here when called with `eps`?

Reply:
No, but I wrote that just for completeness. I removed it to be safe